portals/devportal/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx

```diff
@@ -420,11 +453,30 @@ class ApiConsole extends React.Component {
                 authorizationHeader = 'apikey';
             }
         }
+        let swaggerSpec = swagger;
+        if (api.advertiseInfo && api.advertiseInfo.advertised) {
+            authorizationHeader = advAuthHeader;
+            if (selectedEndpoint === 'PRODUCTION') {
```

Comment:
We need to support Swagger 2.0 definitions as well, It has host property instead of servers

Reply:
Added from https://github.com/wso2/apim-apps/pull/130/commits/c5a27fecad895afea271b84a867305118f05405b